### ISSUE TITLE
Mention limitations

### ISSUE THREAD

**Issue by aleclarson** [alloc/vite-dts#2]
Incompatible with libraries using the following options in **tsconfig.json**...

- `baseUrl`
- `paths`
- `esModuleInterop` (if not also enabled in the consuming project)

<sub>Maybe more?</sub>

**Comment by oliverzy** [comment]
Yes, just hit the 'paths' issue today. I think it's better to document it

**Comment by parkerault** [comment]
I'm also unable to use this plugin because our library uses `typescript-plugin-css-modules`, so if the host application doesn't have it installed the type check fails (it will build fine from the bundle if you don't have a type validation stage in your build toolchain, but `tsc --noEmit` will fail because for some reason it doesn't treat the imports the same as it would from other node_modules packages). If you are using it to build libraries that you will use in your own host applications with the same build toolchain and configuration it's fine, but I don't think it's the right solution for a library meant for public consumption. It would be nice if these limitations were mentioned in the README, as it would have saved me a couple of hours of troubleshooting.
